packages/transaction-controller/src/utils/simulation.ts

```diff
@@ -47,6 +56,41 @@ type ParsedEvent = {
   abi: ABI;
 };
 
+const log = createModuleLogger(projectLogger, 'simulation');
+
+const SUPPORTED_EVENTS = [
+  'Transfer',
+  'TransferSingle',
+  'TransferBatch',
+  'Deposit',
+  'Withdrawal',
+];
+
+const SUPPORTED_TOKENS = {
```

Comment:
SUPPORTED_TOKENS very close to the enum "SupportedTokens"
Should we rename this to something like SUPPORTED_TOKEN_ABIS?

Reply:
Good point, done.